packages/elcollectooorr/skills/elcollectooorr_abci/models.py

```diff
@@ -142,8 +142,8 @@ def _get_decision_model_type(self, kwargs: dict) -> Type[ABC]:
         return valid_types[model_type]
 
 
-class Params(ElCollectooorParams, TransactionParams, FractionalizeDeploymentParams):
```

Comment:
The removal of `TransactionParams` is a side effect of having `TerminationParams` extend `TerminationParams`. 
I think this is more confusing like this, than having the `TransactionParams` be imported as a requirement for abci apps that don't have a direct dependency on `transaction_settlement_abci`. I think that most (not to say all) chained abci apps will have a direct dependency on `transaction_settlement_abci`.

Reply:
Yes, this seems confusing. What would you suggest? Adding the tx settlement dependency back?

Reply:
I think removing it from `TerminationParams` is the best option.

Reply:
I agree. 

What I meant is that it would require adding a dependency back as mentioned in https://github.com/valory-xyz/open-autonomy/pull/1459#discussion_r1011629337.

Reply:
Sorry I misunderstood you, yes that's what I was thinking as well.

Reply:
I am not sure. It deserves further thought before we revert it. Let's track it in an issue first.